README.md

```diff
@@ -4,15 +4,28 @@ Sourcegraph's Phabricator integration adds Sourcegraph code intelligence and sea
 
 ## Installation
 
-1. Clone this repository to `phabricator/src/extensions/sourcegraph` (inside your Phabricator installation directory). See [Phabricator's adding new classes and extensions docs](https://secure.phabricator.com/book/phabcontrib/article/adding_new_classes/) for more information.
+1. Navigate to the Phabricator root directory on your Phabricator instance (eg. `/opt/bitnami/phabricator` if you're using the https://github.com/bitnami/bitnami-docker-phabricator).
+2. Clone this repository to `src/extensions/sourcegraph`:
 
 ```
-git clone -b release-v1.2 https://github.com/sourcegraph/phabricator-extension.git phabricator/src/extensions/sourcegraph
+git clone -b release-v1.2 https://github.com/sourcegraph/phabricator-extension.git src/extensions/sourcegraph
 ```
+3. Run `bin/celerity map` to [add the static CSS/JS assets](https://secure.phabricator.com/book/phabcontrib/article/adding_new_css_and_js/).
```

Comment:
I think you need `cd phabricator ; ./bin/celerity map`

Reply:
All of these steps are actually written so as to be executed from `phabricator`, with step 1 being `Navigate to the Phabricator root directory `